packages/joy-media/src/upload/UploadVideo.tsx

```diff
@@ -93,6 +96,10 @@ const InnerForm = (props: MediaFormProps<OuterProps, FormValues>) => {
     return <em>ERROR: "Video" entity class in undefined</em>
   }
 
+  if (entity && entity.channel && !entity.channel.roleAccount.eq(myAccountId)) {
```

Comment:
There is a utility function `isAccountAChannelOwner`, please use it instead. Don't forget the import :)

`import { isAccountAChannelOwner } from '../channels/ChannelHelpers'`

```suggestion
  if (entity && !isAccountAChannelOwner(entity.channel, myAccountId)) {
```